docs/data_ingestion.asciidoc

```diff
@@ -0,0 +1,103 @@
+[[tune-data-ingestion]]
+= Tune Data Ingestion
+
+[partintro]
+--
+APM Server offers a set of <<configuring-howto-apm-server, configuration options>>,
+helping you to adapt Elastic APM according to your needs.
+
+For optimizing your Elastic APM setup read more about how to:
+
+* <<tune-apm-server>>
+* <<tune-es>>
+
+--
+
+[[tune-apm-server]]
+== Tune APM Server
+You can fine tune your setup considering following options:
+
+* <<tune-output-config>>
+* <<increase-queue-size>>
+* <<increase-concurrent-requests>>
+* <<add-apm-server-nodes>>
+* <<reduce-payload-size>>
+
+[[tune-output-config]]
+[float]
+=== Tune APM Server output parameters for your Elasticsearch cluster
+
+If your Elasticsearch cluster is sized properly,
+but not ingesting the amount of data you expect,
+you can tweak APM Server options to make better use of the cluster:
+
+* Increase `output.elasticsearch.workers` to a number that suits your setup.
+See {ref}/tune-for-indexing-speed.html[tune for indexing speed] for an overview.
+* Ensure `output.elasticsearch.bulk_max_size` is set to a high value, for example 5120.
+  The default of 50 is very conservative.
+* Ensure that `queue.mem.events` is set to a reasonable value compared to your other settings.
+A good rule of thumb is that `queue.mem.events` should equal `output.elasticsearch.worker` multiplied by `output.elasticsearch.bulk_max_size`.
+
+Get detailled information on available <<configuring-output,output configuration options>> and their default values.
+
+[[increase-queue-size]]
+[float]
+=== Increase internal queue size
+
+APM Server uses an internal queue to allow buffering incoming requests until they can be delievered to Elasticsearch. 
+A larger internal queue allows Elasticsearch to be unavailable for longer periods,
+and it alleviates problems that might result from sudden spikes of data.
+You can increase the queue size by overriding `queue.mem.events`.
+Be aware that increasing `queue.mem.events` can significantly affect APM Server memory usage.
+
+[[increase-concurrent-requests]]
+[float]
+=== Increase concurrent requests
+To avoid overfloating the APM Server has an upper limit to how many requests are accepted concurrently.
```

Comment:
A native speaker would have a better recommendation, but "overfloating" doesn't sound right to me. Maybe overwhelming, saturating, overflowing. I also think you need a pronoun somewhere:
"to avoid... the APM Server, it has an..."